dbcollector/src/main/java/com/controller/Main.java

```diff
@@ -0,0 +1,114 @@
+package com.controller;
+
+import com.controller.collectors.DBCollector;
+import com.controller.collectors.PostgresCollector;
+
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Controller main.
+ * @author Shuli
+ */
+public class Main {
+    private static final int DEFAULT_TIME = 5000;  //default observation time: 5000 ms
+    public static void main(String[] args) {
+        // Parse command line argument
+        if(args.length % 2 != 0) {
+            System.out.println("Command line argument mal formed");
+            return;
+        }
+        int time = DEFAULT_TIME; // set time to default
+        for(int i = 0; i < args.length; i += 2){
+            String flag = args[i];
+            String val = args[++i];
+            switch (flag) {
+                case "-t" :
+                    time = Integer.valueOf(val);
+                    break;
+                default:
+                    System.out.println("Invalid flag: " + flag);
+                    break;
+            }
+        }
+
+        // Parse input config file
+        String configFileName = "input_config.json";
+        HashMap<String, String> input = ConfigFileParser.getInputFromConfigFile(configFileName);
+        // db type
+        String dbtype = input.get("database_type");
```

Comment:
Convert the dbtype to lowercase just in case the user uses different casing

Reply:
Are you saying change the string representing db type in input_config.json to all lower case? (done that)